src/components/QuickLinksDialog/components/LinksList/index.js

```diff
@@ -18,19 +18,19 @@ export default class GordonLinksList extends Component {
     return (
       <Typography>
         <List component="nav" subheader={<ListSubheader component="div">Academics</ListSubheader>}>
-          <ListItemLink target="_blank" rel="noopener noreferrer" href="https://www.gordon.edu">
+          <ListItemLink target="_blank" rel="noopener" href="https://www.gordon.edu">
             <ListItemIcon>
               <img
-                className="mygordon"
+                className="gordon"
                 src="https://www.gordon.edu/favicon.ico"
-                alt="my gordon"
+                alt="gordon"
                 width="16"
                 height="16"
               />
             </ListItemIcon>
-            <ListItemText primary="Gordon Main" />
+            <ListItemText primary="Gordon College Official" />
```

Comment:
I'd suggest just "Gordon College" since all of these are official websites of the college.

Reply:
I will make that change in UserViewsEnhancement